_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,294 @@
+---
+name: Iframes with identical accessible names serve equivalent purpose
+
+description: |
+ This rule checks that iframes with identical accessible names serve an equivalent purpose
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to `iframe` elements that are [included in the accessibility tree](#included-in-the-accessibility-tree) and have [matching](matching-characters) [accessible name](#accessible-name), that do not only include whitespace.
+
+**Note**: As the `frame` element is deprecated, this rule does not consider `frame` or `frameset` elements.
+
+### Expectation
+
+Iframes with identical [accessible name](#accessible-name) in the same [document tree](https://www.w3.org/TR/dom41/#document-trees) serve [equivalent purpose](#equivalent-purpose).
```

Comment:
The definitions in https://github.com/auto-wcag/auto-wcag/pull/220 has changed due to reviews since I wrote the comments that you just worked in, so I would suggest something like this: 

> The `iframes` in each set of target elements embeds the the [same resource](#same-resource) or [equivalent resources](#equivalent-purpose).